fixtures/manufacturers.json

```diff
@@ -433,6 +433,9 @@
     "website": "https://sgmlight.com/",
     "rdmId": 21319
   },
+  "shehd": {
+    "name": "SHEHD"
+  },
```

Comment:
```suggestion
```

This should obviously be in `shehds/`, not `shehd/` -- I don't know how to suggest moving a file from a review.